indexer/packages/postgres/src/db/migrations/migration_files/20240809153326_create_firebase_notification_tokens_table.ts

```diff
@@ -0,0 +1,16 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable('firebase_notification_tokens', (table) => {
+    table.increments('id').primary();
+    table.string('token').notNullable().unique();
+    table.string('address').notNullable();
+    table.foreign('address').references('wallets.address').onDelete('CASCADE');
```

Comment:
I'm not sure the `onDelete('CASCADE')` is needed here. It doesn't seem to be present on other foreign keys defined in other tables.

Reply:
This is basically saying that if we delete an wallet address the DB should automatically delete any tokens that reference that address. Which is the behavior we want imo.